### PR TITLE
Add a plugin that allows copying .cat and .inf files from the source …

### PR DESCRIPTION
**Pull request by hughsie**
…archive

This allows a vendor to use the 'output' of Windows Update as the 'input' to
the LVFS and have one archive that can be used on either OS.

Fixes https://github.com/hughsie/lvfs-website/issues/40

### CODE REVIEW

**Comment by superm1** [comment]
Just to play devil's advocate, why not just copy all files? It would be unfortunate if Microsoft introduced some new extension they used and we didn't catch it for a while etc.

**Comment by hughsie** [comment]
> why not just copy all files

Several reasons:

 * We want to tightly control the files provided by the LVFS, in case a vendor included a nonfree `flash.exe` -- the legal advice we've got on this makes it clear that shipping this kind of not-explicitly-redistributable executable binary would make things somewhat more icky. I'm happy enough to argue that the .inf file was created by the vendor for distribution without a EULA, and that there's no inherent license on a .cat file as it's the binary output of a reproducible signing procedure.

 * We want to force the vendor to put the release notes in the metainfo file (or using the web interface) rather than build a `Release Nores.doc` file that's not designed to be user or machine readable.

 * We want to keep the size of the files as small as possible; more than one vendor shipped different variants of the firmware in the .cab archive, referencing only one in the metainfo file. By stripping out anything not in the metainfo that's a pretty easily understood case where the vendor actually needs more than one metainfo file in the .cab file, with a different GUID and appstream `<id>`.

**Comment by superm1** [comment]
OK, make sense to me 👍

File: plugins/wu-copy/__init__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python2
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+# Licensed under the GNU General Public License Version 2
+
+from __future__ import print_function
```
Comment (superm1) [inline plugins/wu-copy/__init__.py]: You don't actually use the print function function in this plugin

Comment (hughsie) [reply plugins/wu-copy/__init__.py]: Fixed, thanks. I wonder why pylint doesn't catch unused imports in files named `__init__.py`...

**Pull request merged**